CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 ## x.y.z 2023-xx-yy
 ### PaymentSheet
-* [Added] Enable SEPA Debit, bancontact, iDEAL for SetupIntents and PaymentIntents with setup_future_usage. Note: PaymentSheet doesn't display saved SEPA Debit payment methods yet.
+* [Added] Enable SEPA Debit, bancontact, iDEAL, sofort for SetupIntents and PaymentIntents with setup_future_usage. Note: PaymentSheet doesn't display saved SEPA Debit payment methods yet.
```

Comment:
Can we get the latest changes from master, I think the changelog will get a little messed up.